sticker/src/tensorflow/lr.rs

```diff
@@ -8,7 +8,9 @@ use std::f32;
 /// at a given epoch.
 pub trait LearningRateSchedule {
     /// Compute the learning rate for an epoch.
-    fn learning_rate(&mut self, epoch: usize, last_score: f32) -> f32;
+    fn compute_epoch_learning_rate(&mut self, epoch: usize, last_score: f32) -> f32;
+    /// Compute the learning rate for the current batch.
+    fn compute_step_learning_rate(&mut self, global_step: usize) -> f32;
```

Comment:
Changed this from `compute_batch_learning_rate` to `compute_batch_learning_rate` to be consistent with `global_step` and `warmup_steps`.